src/cursor/cur_backup.c

```diff
@@ -348,6 +348,9 @@ __wt_curbackup_open(WT_SESSION_IMPL *session, const char *uri, WT_CURSOR *other,
     if (othercb != NULL)
         WT_CURSOR_BACKUP_CHECK_STOP(othercb);
 
+    if (cfg != NULL && cfg[1] != NULL)
+        __wt_verbose(session, WT_VERB_BACKUP, "Backup cursor config \"%s\"", cfg[1]);
```

Comment:
I'm fine with the if statement either way. I originally suggested it in the ticket because backup verbose is rarely on and opening a backup cursor is rare to begin with. But opening a backup cursor is not a performant path that calling wt_verbose every time to discover it is not turned on does not matter.